data/ref-cube.mesh

```diff
@@ -11,6 +11,21 @@ MFEM mesh v1.0
 # CUBE        = 5
 # PRISM       = 6
 #
+#       7-------6
+#      /|      /|
+#     / |     / |
+#    /  |    /  |
+#   3-------2   |
+#   |   4---|---5
+#   |  /    |  /
+#   | /     | /
+#   |/      |/
+#   0-------1
+#
+#   y
+#   | z
+#   |/
+#   *--x
```

Comment:
I have always thought of the axes as

```suggestion
#   z
#   | y
#   |/
#   *--x
```

and there may be places in the code that already use that convention. (This is also the "default" in GLVis)

Reply:
I actually did the axes differently in a couple different places because I was having trouble with the ascii art 😅.  Let me see if I can turn them and still get the art right.  Any good ASCII artists out there?

Reply:
Can't say I am any good, but I will be certainly happy to try to help 😁 

@jakubcerveny has some similar diagrams here:

https://github.com/mfem/mfem/blob/master/mesh/ncmesh.cpp#L1321-L1332

```c++
// Tetrahedron vertex numbering:
//
//    3
//     +                         Faces: 0 back (1, 2, 3)
//     |\\_                             1 left (0, 3, 2)
//     ||  \_                           2 front (0, 1, 3)
//     | \   \_                         3 bottom (0, 1, 2)
//     |  +__  \_
//     | /2  \__ \_       Z  Y
//     |/       \__\      | /
//     +------------+     *--X
//    0              1
```

https://github.com/mfem/mfem/blob/master/mesh/ncmesh.cpp#L931-L943

```c++
// Vertex numbering is assumed to be as follows:
//
//       7             6
//        +-----------+                Faces: 0 bottom
//       /|          /|                       1 front
//    4 / |       5 / |                       2 right
//     +-----------+  |                       3 back
//     |  |        |  |                       4 left
//     |  +--------|--+                       5 top
//     | / 3       | / 2       Z Y
//     |/          |/          |/
//     +-----------+           *--X
//    0             1
```

https://github.com/mfem/mfem/blob/master/mesh/ncmesh.cpp#L1196-L1208

```c++
// Wedge vertex numbering:
//
//          5
//         _+_
//       _/ | \_                    Faces: 0 bottom
//    3 /   |   \ 4                        1 top
//     +---------+                         2 front
//     |    |    |                         3 right (1 2 5 4)
//     |   _+_   |                         4 left (2 0 3 5)
//     | _/ 2 \_ |           Z  Y
//     |/       \|           | /
//     +---------+           *--X
//    0           1
```

Here is an example from hypre: https://github.com/hypre-space/hypre/blob/master/src/examples/ex15.c#L495-L509

```c++
int ordering[48] =
{
    0,  0, -1, -1,    /* x-edge [0]-[1]                  */
    1, +1,  0, -1,    /* y-edge [1]-[2]                  */
    0,  0, +1, -1,    /* x-edge [3]-[2]     [7]------[6] */
    1, -1,  0, -1,    /* y-edge [0]-[3]     /|       /|  */
    0,  0, -1, +1,    /* x-edge [4]-[5]    / |      / |  */
    1, +1,  0, +1,    /* y-edge [5]-[6]  [4]------[5] |  */
    0,  0, +1, +1,    /* x-edge [7]-[6]   | [3]----|-[2] */
    1, -1,  0, +1,    /* y-edge [4]-[7]   | /      | /   */
    2, -1, -1,  0,    /* z-edge [0]-[4]   |/       |/    */
    2, +1, -1,  0,    /* z-edge [1]-[5]  [0]------[1]    */
    2, +1, +1,  0,    /* z-edge [2]-[6]                  */
    2, -1, +1,  0     /* z-edge [3]-[7]                  */
};
```

Reply:
Thanks for the examples. I think I managed to cobble something together.  If only ascii has some 30 and 60 degree angles as well...